eigen_kdl/package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>eigen_kdl</name>
```

Comment:
Since this now adds `tf2` conversion messages to this code, I'll suggest we rename this to `tf2_eigen_kdl` (this also fits the pattern of the rest of the packages in this repository).

@tfoote What do you think?

Reply:
Yeah that makes sense.